js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadTransientSessionContainer.tsx

```diff
@@ -42,9 +42,11 @@ export const LaunchpadTransientSessionContainer = (props: Props) => {
     rootDefaultYaml,
     !flagDisableAutoLoadDefaults,
   );
+
+  // Avoid supplying an undefined `runConfigYaml` to the session.
   const initialSessionComplete = createSingleSession({
     ...sessionPresets,
-    runConfigYaml: initialData.runConfigYaml,
+    ...(initialData.runConfigYaml ? {runConfigYaml: initialData.runConfigYaml} : {}),
```

Comment:
Wow this is subtle - good catch 🙌  I wonder if there's a lodash helper for cases like this that does a "merge truthy values"... feels like this is better because it's more explicit what is happening.